packages/ui/src/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
do we need this?

Reply:
they are automatically generated by vite, i understand that they are needed for IntelliSense to work with custom env variables https://vitejs.dev/guide/env-and-mode.html#intellisense-for-typescript